### PR TITLE
[Impeller] Interpret odd path winding in the dispatcher

### PR DESCRIPTION
**Pull request by bdero**
`impeller_unittests --gtest_filter=Play/DisplayListTest.CanDrawWithOddPathWinding/Metal --timeout=0`:
image

### CODE REVIEW

File: impeller/display_list/display_list_dispatcher.cc [hunk 1]
```diff
@@ -490,8 +491,22 @@ static Path ToPath(const SkPath& path) {
...
+  switch (path.getFillType()) {
+    case SkPathFillType::kWinding:
+      fill_type = FillType::kNonZero;
+      break;
+    case SkPathFillType::kEvenOdd:
+      fill_type = FillType::kOdd;
+      break;
+    case SkPathFillType::kInverseWinding:
+    case SkPathFillType::kInverseEvenOdd:
+      UNIMPLEMENTED;
```
Comment (zanderso) [inline impeller/display_list/display_list_dispatcher.cc:505]: Is this something that we'll eventually need? If so let's add a TODO and file an issue.

**Review by zanderso — APPROVED** [review]
lgtm w/ question

File: impeller/display_list/display_list_dispatcher.cc (hunk 1 above)
Comment (bdero) [reply impeller/display_list/display_list_dispatcher.cc:506]: Done: https://github.com/flutter/flutter/issues/104848

Comment (dnfield) [reply impeller/display_list/display_list_dispatcher.cc:506]: FWIW, Flutter does not expose these methods - https://api.flutter.dev/flutter/dart-ui/PathFillType.html only does evenOdd and nonZero.

Comment (dnfield) [reply impeller/display_list/display_list_dispatcher.cc:506]: (and nonZero == kWinding)

**Pull request merged**